accounts/models.py

```diff
@@ -260,3 +260,12 @@ def create_user_profile(sender, instance, created, **kwargs):
         profile.save()
 
 post_save.connect(create_user_profile, sender=User)
+
+
+class Donation(models.Model):
+    email = models.CharField(max_length=255)
```

Comment:
Now we allow to donate without been logged in and we don't link the donation to the profile, this email is the one returned by paypal after the donation. Right now I only pass display_name to paypal. The only problem is that there's only one parameter to pass extra data so if we want to pass also the id I would have to encode both attributes together.
(first I wanted to know if this was needed)